content/release-10-18-0/devices-10-18-0.md

```diff
@@ -3,3 +3,95 @@ weight: 50
 title: Device management & connectivity
 layout: bundle
 ---
+
+<!--10.17.1.0 - 10.17.100.0-->
+
+### Improvements
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 55%;"><col style="width: 15%;"><col style="width: 15%;"></colgroup><thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+<th>
+Fix Version</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Device Management</td>
+<td>Added support for rendering dropdown fields in Formly from JSON Schema in the extensible device registration.</td>
```

Comment:
maybe "Formly JSON Schema" instead of "Formly from JSON Schema"

Reply:
Would that be the same?

Reply:
i'm not sure about this one
Formly is an AngularJS library, it seems: https://formly.dev/
and on the site it says that it supports multiple schemas (Formly schema and JSON schema, so they are distinct)
i don't know which one is meant here, could be either. who could tell us?

Reply:
From the ticket I´d say it is JSON Schema Formly. This is actually what it says. Formly from JSON Schema (as opposed to AngularJS).